app/views/companies/show.html.haml

```diff
@@ -1,51 +1,49 @@
-%header.entry-header
-  %h1.entry-title= @company.name
-  .post-title-divider
+%div{ :id => "#{@company.class.name.downcase}-#{@company.id}", class: "show-item #{@company.class.name.downcase} #{@company.class.name.downcase}-#{@company.id}" }
```

Comment:
just to document our decision from our pair programming:  I'll write some helper methods that we can later expand and/or use.